### PR TITLE
Add note to clarify item groups lack of nesting limit

### PR DESCRIPTION
**Pull request by jcollado**
In this PR, a note is added to make clear that there's no limit in how nested the item groups structure can be.

[ch825]

### CODE REVIEW

File: source/includes/_item_groups.md [hunk 1]
```diff
@@ -24,6 +24,10 @@ id | string | Item group customer ID.
 name | string | Item group display name.
 children | List[item_group] | List of item groups categorized under the current item group.
 
+**Note:** there is a limit in the number of item groups that can be submitted
```
Comment (adetch) [inline source/includes/_item_groups.md]: Could you note what the limit is set at?

Comment (jcollado) [reply source/includes/_item_groups.md]: Fixed in b66ced0

**Comment by jcollado** [comment]
@arthur Added the limit to the documentation. Please let me know if there's any other issue.

File: source/includes/_item_groups.md [hunk 2]
```diff
@@ -24,6 +24,11 @@ id | string | Item group customer ID.
...
```
Comment (adetch) [inline source/includes/_item_groups.md]: I'd just suggest shortening this up.
> **Note:** Up to 1,000 item groups can be submitted in a single request. However, there is no limit to the depth of nested groups within an item group.

**Pull request merged**